ebib.el

```diff
@@ -4179,7 +4179,11 @@ value from a cross-referenced entry, it is not killed."
   (let ((key (ebib--get-key-at-point)))
     (unless (or (not field)
                 (string= field "=type="))
-      (let ((contents (ebib-get-field-value field key ebib--cur-db 'noerror 'unbraced 'xref)))
+      (let ((contents (if (string= field "external")
+			  (with-current-buffer
+			      (find-file-noselect (ebib--create-notes-file-name key))
+			    (buffer-string))
+			(ebib-get-field-value field key ebib--cur-db 'noerror 'unbraced 'xref))))
```

Comment:
This doesn't look very robust in the case a user has multiple notes per file, right?

Come to think of it, `ebib-kill-field-contents` and `ebib-delete-field-contents` should really be one single function with an optional argument indicating whether the deleted content should be added to the kill ring or not. If it should, `ebib-copy-field-contents` should probably be used to get the field contents and place it in the kill ring. That way, any improvements added to handle the "external note" field would only have to be made in one place.

It would be fine to only handle the `one-file-per-note` case first, esp. for killing/deleting. (It should be easier to implement copying for the `multiple-note-per-file` case, though. The function `ebib--extract-note-text` could be a starting point,  if it can be modified to only optionally truncate the note text.)